transport/http2_client.go

```diff
@@ -232,6 +234,24 @@ func newHTTP2Client(ctx context.Context, addr TargetInfo, opts ConnectOptions) (
 		kp:              kp,
 		statsHandler:    opts.StatsHandler,
 	}
+	t.bdp = &bdpEstimator{
+		p: &ping{
+			data: [8]byte{1, 2, 3, 4, 5, 6, 7, 8},
+		},
+		estimate:  initialConnWindowSize,
+		maxWindow: maxWindowSize,
+		send: func(p *ping) {
+			t.controlBuf.put(p)
+		},
+		updateConnWindow: func(delta uint32) {
+			// Update the limit on fc.
+			t.fc.mu.Lock()
+			t.fc.limit += delta
+			t.fc.mu.Unlock()
+			// Send a window update on the connection.
+			t.controlBuf.put(&windowUpdate{0, delta})
```

Comment:
Is this actually enough to increase the window persistently? I'd expect there'd be some value for the "target" window that needs to be increased as well.